mod_reforged/hooks/skills/skill_container.nut

```diff
@@ -0,0 +1,22 @@
+::Reforged.HooksMod.hook("scripts/skills/skill_container", function(q) {
+	q.onDamageReceived = @(__original) function( _attacker, _damageHitpoints, _damageArmor )
+	{
+		local damage = _damageArmor + ::Math.min(_damageHitpoints, this.getActor().getHitpoints());
+
+		__original(_attacker, _damageHitpoints, _damageArmor);
+
+		if (_attacker != null)
```

Comment:
We should also make sure that `damage != 0` here. Otherwise we write `0.0` in various variables which could later cause a divided by 0

Reply:
Good catch. The PR was already merged by this point. I fixed this in the commit on development branch and force pushed that.